src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2428,6 +2428,11 @@
     crafted: true
 """
 
+helps['vmss extension upgrade'] = """
+type: command
+short-summary: Upgrade all extensions for all VMSS instances to the latest version.
+"""
```

Comment:
Add at least one example for one command?

Reply:
Added.